lib/ansible/plugins/action/__init__.py

```diff
@@ -160,7 +160,7 @@ def _configure_module(self, module_name, module_args, task_vars=None):
         '''
 
         if task_vars is None:
```

Comment:
not sure this is needed either, its only called from 2 locations and they always pass task_vars .. i would make it a required field at this point

Reply:
I went ahead and made the `task_vars` required method argument in this PR.